packages/synthetix-main/hardhat.config.js

```diff
@@ -8,6 +10,10 @@ module.exports = {
     local: {
       url: 'http://localhost:8545',
     },
+    kovan: {
```

Comment:
I like this more than how its done in v2x.

Instead of manually setting up provider and signer all over the place, you just do it here in the hardhat config file.
The provider will be available in the hre, as well as the first signer.